src/restconfig/src/test/java/org/geoserver/rest/catalog/LayerGroupControllerTest.java

```diff
@@ -23,12 +18,28 @@
 import org.geotools.referencing.CRS;
 import org.junit.Before;
 import org.junit.Test;
-import org.w3c.dom.Document;
-
 import org.springframework.mock.web.MockHttpServletResponse;
+import org.w3c.dom.Document;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
+import static org.custommonkey.xmlunit.XMLAssert.assertXpathEvaluatesTo;
```

Comment:
IntelliJ vs Eclipse clash detected :-p (Eclipse puts static imports at top, I guess IntelliJ does the opposite?)